MANIFEST.in

```
@@ -1,3 +1,5 @@
 include LICENSE.txt
 include requirements.txt
 include qiskit_experiments/VERSION.txt
+include qiskit_experiments/library/randomized_benchmarking/data/*.npz
+exclude qiskit_experiments/library/randomized_benchmarking/data/*.py
```

```suggestion
exclude qiskit_experiments/library/randomized_benchmarking/data/generate_clifford_data.py
```
I think you must explicitly state the file name. Otherwise this could cause unexpected bug in the future update.

---

I've found the simplest way to include *.npz and exclude *.py is do nothing here (and having no init file for the data directory). Resolved in a04310a